second_pseudo_code_on_right_angle_symbol.md

```diff
@@ -0,0 +1,38 @@
+# Pseduo code
+
+## Define the following function
+
+const RightAngleSymbol (pt0,pt1,pt2)
```

Comment:
Place codes within triple back-ticks can make Markdown recognize that this is a block of code.

E.g.

```javascript
const a = 1
console.log(a)
```